content/error-handling/index.textile

```diff
@@ -0,0 +1,130 @@
+
+All errors returned by Ably are standardized and use the @ErrorInfo@ object:
+
+```[text]
+{
+  code: 40005,
+  statusCode: 400,
+  cause: Authentication,
+  nonfatal: false,
+  href: 'https://help.ably.io/error/40005'
+}
+```
+
+The following explains each property of an @ErrorInfo@ object:
+
+- *code* := Ably-specific numeric code indicating the error type.
+- *statusCode* := An HTTP status code providing broader context, such as 400 for a bad request.
+- *cause* := A brief description of the issue.
+- *nonfatal* := A boolean indicating whether the error is critical.
+- *Href* := A direct link to Ably's documentation or for quick troubleshooting references.
+
+h2(#logging). Logging
+
+Most Ably SDKs allow you to customize the function that handles logging. This function is usually set in the options when configuring a client, such as the @ClientOptions@ object for Pub/Sub.
+
+Two separate properties can be set:
+
+- *logHandler* :=  Provide a custom function for each line of log output.
+- *logLevel* := The verbosity of the logging output, from silent to trace. In some SDKs this is numeric, and in others string.
+
+The following example configures the Ably client to log only error messages (level 1) and processes them with the function @logWriteFun()@:
+
+```[text]
+const ablyClient = new AblyClient(ably, {
+  logHandler: logWriteFunc,
+  logLevel: 1 // Errors only
+});
+```
+
+The following is an example log entry:
+
+```[text]
+LOG [Level 1]: Ably: ConnectionManager.failQueuedMessages(): failing 1 queued messages, err = [_ErrorInfo: Application Ptz0jg disabled.
+(See https://help.ably.io/error/40300 for help.); statusCode=403; code=40300]
+```
+
+h2(#meta). [meta]log channels
+
+Errors are not published in the meta channels when the client can be directly notified. For example, if a client publishes a message but exceeds a channel rate limit, the @publish()@ method uses an error callback to notify the client.
+
+There are two @metachannels@ related to logging:
+
+- @Meta[log]@ := Publishes errors that aren't visible to clients, except those related to push notifications.
+
+- @Meta[log:push]@ := Similar to @[meta]log@, but only publishes errors related to the delivery of push notifications.
+
+The following example subscribes to the @[meta]log@ channel:
+
+```[text]
+const channel = realtime.channels.get('[meta]log');
+channel.subscribe((msg) => {
+  console.log(msg);
+});
+```
+
+The following is an example event published to the @[meta]log@ channel as an @ErrorInfo@ object:
+
+```[text]
+{
+  code: 40005,
+  statusCode: 400,
+  cause: Authentication,
+  nonfatal: false,
+  href: 'https://help.ably.io/error/40005'
+}
+```
+
+h2(#console). Dev console
+
+The Dev console, accessible from your Ably dashboard, is essential for inspecting and debugging realtime errors. It visually displays all events in your application, making it particularly valuable during early-stage development or low-traffic periods when event tracking is simpler:
```

Comment:
I wouldn't say it's essential... I think we should also note that it's a live, or realtime stream of events so that it's very clear that you can't query it after an error has occurred.